new/LIT7125QeneRufael.xml

```diff
@@ -0,0 +1,76 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7125QeneRufael" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሩፋኤል፡ ላዕከ፡ ተሰብኦ፡ ባሕረ፡ ተሰብኦ፡ አመ፡ ጸበተ።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">Rufāʾel lāʿka tasabʾo bāḥra tasabʾo ʾama ṣabata... (Qǝne possibly of =sǝllāse type)</title>
```

Comment:
It may be a dittography, but possibly not. The line may mean: "Rufael, the servant of incarnation, when he swam in the see of the incarnation..." 
The verses have rhyme on ለ. This may be a sellase qene, as you indicated.